recipes/gapseq/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = "1.2" %}
+
+package:
+  name: gapseq
+  version: {{ version }}
+
+source:
+  url: https://github.com/jotech/gapseq/archive/refs/tags/v{{ version}}.tar.gz
+  sha256: 061799f8e0406662450dd3d8d61d824220670fb23231895deecd490a5cbdf7cf
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('gapseq', max_pin="x") }}
+  skip: True # [osx]
+
+requirements:
+  run:
+    - r-base =4.3 # Version specification is not strictly necessary, but can help avoiding future sudden problems due to incompatible changes to R.
+    - perl
+    - parallel
+    - gawk
+    - sed
+    - grep
+    - bc
+    - git
+    - coreutils
+    - wget
+    - openssl
+    - barrnap
+    - bedtools
+    - exonerate
+    - glpk
+    - hmmer
+    - blast
+    - bioconda::libsbml 
+    - r-data.table
+    - r-stringr
+    - r-stringi
+    - r-getopt
+    - r-doParallel
+    - r-foreach
+    - r-r.utils
+    - r-sybil
+    - r-biocmanager
+    - bioconductor-biostrings
+    - r-jsonlite
+    - r-renv
+    - r-glpkapi
+    - r-rcurl
+    - r-httr
+    - r::r-chnosz # Is available on bioconda directly
+  build: 
+    - wget
+
+test:
+  commands:
+    - gapseq test
+
+about:
+  home: https://github.com/jotech/gapseq
+  summary: Informed prediction and analysis of bacterial metabolic pathways and genome-scale networks 
+  license_family: GPL
+  license: GPL-3.0
+  license_file: LICENSE
+extra:
+  skip-lints:
+    - should_be_noarch_generic
```

Comment:
why is it not noarch: generic?

Reply:
Because the linter gives this error, even though it is not noarch_generic (doesn't work on macos).